src/app/work-item/work-item-quick-add/work-item-quick-add.component.ts

```diff
@@ -1,21 +1,22 @@
-import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
-import { ActivatedRoute, Params } from '@angular/router';
+import { Component, EventEmitter, Input, OnInit, Output } from "@angular/core";
+import { ActivatedRoute, Params } from "@angular/router";
 
-import { Logger } from '../../shared/logger.service';
-import { WorkItem } from '../work-item';
-import { WorkItemService } from '../work-item.service';
+import { Logger } from "../../shared/logger.service";
+import { WorkItem } from "../work-item";
+import { WorkItemService } from "../work-item.service";
```

Comment:
I agree. I was just following the test report. We can remove these commits if not needed.